tests/Unit/Symfony/OtelSdkBundle/DependencyInjection/Configuration/Validator/CustomClassImplementationValidatorTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Test\Unit\Symfony\OtelSdkBundle\DependencyInjection\Configuration\Validator;
+
+use Exception;
+use OpenTelemetry\Symfony\OtelSdkBundle\DependencyInjection\Configuration\Validator\CustomClassImplementationValidator;
+use OpenTelemetry\Symfony\OtelSdkBundle\DependencyInjection\ConfigurationException;
+use PHPUnit\Framework\TestCase;
+use stdClass;
+use Throwable;
+
+/**
+ * @covers \OpenTelemetry\Symfony\OtelSdkBundle\DependencyInjection\Configuration\Validator\CustomClassImplementationValidator
+ */
+class CustomClassImplementationValidatorTest extends TestCase
+{
+    public function test_approve(): void
+    {
+        CustomClassImplementationValidator::approve(
+            Exception::class,
+            Throwable::class
+        );
+
+        $this->assertInstanceOf(Throwable::class, new Exception());
```

Comment:
this looks weird, is it really just a dummy assertion?

Reply:
This method is throwing an exception, if a class does not implements a certain interface, so it looks like a dummy assertion, but It's not. However the validators do not return anything, these are sinply guard methods.
The downside of `expectNotToPerformAssertions` is also that it removes the code from the coverage report.